### PR TITLE
Add theme for components

### PR DESCRIPTION
**Pull request by Karthik-B-06**
[chore(avatar): 💄 update active status bg color](https://github.com/adaptui/react-native-tailwind/commit/cfaebb8b27da32446f7510ce37884fc2b4d19981)

[chore(badge): 🎨 add theme and set default theme to base](https://github.com/adaptui/react-native-tailwind/commit/fe35738e76b0d63341c29d3f6c6313332c06ccff)

[chore(button): 🎨 add button theme, add themeColor prop & set to base](https://github.com/adaptui/react-native-tailwind/commit/a9e247033f66c7ceeedeaa858782bc11b966cb48)

### CODE REVIEW

**Review by navin-moorthy — COMMENTED** [review]
prefer active styles for pressed state

File: src/components/checkbox/CheckboxGroup.tsx [hunk 1]
```diff
@@ -62,7 +67,7 @@ const RNCheckboxGroup: React.FC<Partial<CheckboxGroupProps>> = forwardRef<
           <Box
             key={index}
             style={tailwind.style(
-              checkboxGroupTheme.group[orientation].spacing,
+              cx(checkboxGroupTheme.group[orientation].spacing),
```
Comment (navin-moorthy) [inline src/components/checkbox/CheckboxGroup.tsx:70]: Add `?.` for orientation.

File: src/theme/defaultTheme/spinner.ts [hunk 2]
```diff
@@ -1,8 +1,21 @@
 export const spinner = {
-  base: "border-2 border-solid rounded-full border-gray-800",
+  base: "border-2 border-solid rounded-full",
```
Comment (navin-moorthy) [inline src/theme/defaultTheme/spinner.ts:2]: border-[1.5px]

Comment (Karthik-B-06) [reply src/theme/defaultTheme/spinner.ts:2]: Added 
```
{ borderWidth: 1.5 },
```

File: src/components/checkbox/CheckboxGroup.tsx [hunk 3]
```diff
@@ -53,7 +58,7 @@ const RNCheckboxGroup: React.FC<Partial<CheckboxGroupProps>> = forwardRef<
   const validChildren = getValidChildren(children);
   return (
     <Box
-      style={tailwind.style(checkboxGroupTheme.group[orientation].common)}
+      style={tailwind.style(cx(checkboxGroupTheme.group[orientation].common))}
```
[the remark this answers is not in the document]
Comment (Karthik-B-06) [reply src/components/checkbox/CheckboxGroup.tsx:61]: Done :tada:

**Pull request merged**